examples/viewer.py

```diff
@@ -1036,15 +1122,49 @@ def next_frame() -> None:
         action="store_true",
         help="Override configured lighting to use synthetic lighting for the stage.",
     )
+    parser.add_argument(
+        "--enable_batch_renderer",
```

Comment:
```suggestion
        "--enable-batch-renderer",
```

I think this is more consistent with option naming everywhere else. Same below.